app/assets/stylesheets/components/tabs.scss

```diff
@@ -50,6 +51,24 @@
       margin: 0 var(--su-1);
     }
 
+    /* 
+      Add a, hidden, bold version of the text to get width.
+      This prevents the minor layout shift on tab change.
+    */
+    &::before {
+      content: attr(data-text);
```

Comment:
Nitpick (non-blocking): We could use `position: absolute` instead of `height: 0`